proxy/rust-toolchain

```diff
@@ -1 +1 @@
-nightly-2020-02-05
+nightly-2020-06-09
```

Comment:
This will need a docker image update for CI, right?

Reply:
See 3aa9fcc